slack/src/main.rs

```diff
@@ -41,6 +41,14 @@ lazy_static! {
         .timeout(Duration::from_secs(TIMEOUT))
         .build()
         .expect("Can't build the reqwest client");
+    static ref EVENTS: Value = {
```

Comment:
It's a good idea, but the Json content can be passed directly to the client.

Reply:
I'll explain, there is a problem:

if pass json content directly as Json() params, the response just a raw string. any suggestions to solve it without serde?

Reply:
In fact, you don't need to go through Json(), return Json string directly and don't forget to modify the Content-Type header.

Reply:
you are right, please review new commit